src/squlearn/qnn/qnnc.py

```diff
@@ -272,7 +274,7 @@ def partial_fit(self, X: np.ndarray, y: np.ndarray, weights: np.ndarray = None)
                 )
         self._is_fitted = True
 
-    def _fit(self, X: np.ndarray, y: np.ndarray, weights: np.ndarray = None) -> None:
+    def _fit(self, X, y, weights: np.ndarray = None) -> None:
```

Comment:
Add doc strings from the fit function array-like or sparse matrix of shape (n_samples, n_features) etc